src/component/mxgraph/renderer/StyleComputer.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Shape from '../../../model/bpmn/shape/Shape';
+import Edge from '../../../model/bpmn/edge/Edge';
+import Bounds from '../../../model/bpmn/Bounds';
+import { ShapeBpmnBoundaryEvent, ShapeBpmnEvent, ShapeBpmnSubProcess } from '../../../model/bpmn/shape/ShapeBpmnElement';
+import { StyleIdentifier } from '../StyleUtils';
+import { Font } from '../../../model/bpmn/Label';
+
+export class StyleComputer {
+  computeStyle(bpmnCell: Shape | Edge, labelBounds: Bounds): string {
```

Comment:
It's close to StyleConfigurator. Maybe, we can merge those 2 classes.

Reply:
I didn't see that opportunity before you mention it.
I see a big advantage in merging the 2 classes: default and dynamic styles would be at the same place so it would be easier to ensure consistency.
The caveats would be a large class (around 250 lines) but this is something acceptable, so go for it!